mplexporter

```diff
@@ -1 +1 @@
-Subproject commit 0168ffcab112d3ab3b60de3042c242f6279ac9a4
+Subproject commit 2e7f21125d2eb45290e6632cab1d26e43204cf43
```

Comment:
@jakevdp : do you have any theory of when the subproject commit should and should not change?